### PR TITLE
Update vectors.md

### PR DESCRIPTION
**Pull request by Sentious**
# Added `.front()` and `.back()` functions for vectors

### Description
Added two functions for vectors `.front()` and `.back()` in vectors section of C++ docs.

### Type of Change

- Adding a new entry
~- Editing an existing entry (fixing a typo, bug, issues, etc)~
~- Updating the documentation~

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Sentious** [comment]
@caupolicandiaz hey can you please review the PR, its been pending for three days without approval.

File: content/cpp/concepts/vectors/vectors.md
Comment (caupolicandiaz) [inline content/cpp/concepts/vectors/vectors.md:100]:
```suggestion
std::cout << order.front();
```

File: content/cpp/concepts/vectors/vectors.md
Comment (caupolicandiaz) [inline content/cpp/concepts/vectors/vectors.md:109]:
```suggestion
std::cout << order.back();
```

File: content/cpp/concepts/vectors/vectors.md
Comment (caupolicandiaz) [reply content/cpp/concepts/vectors/vectors.md:100]: Since this is adding to the section "getting information" which uses a vector declared in the first example throughout all the code blocks, it should be carried over here as well

File: content/cpp/concepts/vectors/vectors.md
Comment (caupolicandiaz) [inline content/cpp/concepts/vectors/vectors.md:96]:
```suggestion
Alternatives to the `.at()` method include dedicated methods for retrieving the first or the last element of a vector. The `.front()` returns a reference to the first element of the vector:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Sentious I've added my comments. Please update the entry, and tag me when you've made the changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@Sentious This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@Sentious Looks good.

**Pull request merged**
